owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/FileListAdapter.kt

```diff
@@ -250,16 +249,25 @@ class FileListAdapter(
 
                 ViewType.GRID_IMAGE.ordinal -> {
                     val view = holder as GridImageViewHolder
+                    val layoutParams = fileIcon.layoutParams as ViewGroup.MarginLayoutParams
+
                     if (thumbnail == null) {
                         view.binding.Filename.text = file.fileName
+                        //Reset layout params values default
```

Comment:
```suggestion
                        // Reset layout params values default
```
Correct format of comments is separating `//` from the message of the commit. But real correct format for comments is not writing comments 😉, because that means the code is not clear enough. In this case I think it's ok, but just FYI for the future 👍